src/SDL_utils.c

```diff
@@ -161,6 +161,48 @@ SDL_bool SDL_ObjectValid(void *object, SDL_ObjectType type)
 void SDL_SetObjectsInvalid(void)
 {
     if (SDL_objects) {
+        SDL_assert(SDL_HashTableEmpty(SDL_objects));
```

Comment:
Perhaps move this assertion after the logging?

Reply:
Good idea, done!